server/main.py

```diff
@@ -1,91 +1,297 @@
 from fastapi import FastAPI
 from fastapi import HTTPException
-
+import uvicorn
+from typing import Optional
+from pydantic import BaseModel
 import storage
 import schemas
+import json
 
 
 app = FastAPI()
 
 
-@app.get("/list-games/")
-async def list_games():
-    return {
-        "games": storage.get_games(),
-    }
+# @app.post("/register-as-a-player/")
+# async def register_as_a_player(player_id_1: str, player_id_2: str):
+#     return storage.register_player(player_id_1, player_id_2)
+###########################################################
+###########################################################
+_GAMES_STORAGE = {
```

Comment:
You already have `_GAMES_STORAGE` in `storage.py`. I didn't get why you created it here additionally.